core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -750,6 +784,27 @@ public void responseContent(@Nullable Object responseContent, @Nullable Object r
         updateAvailability(RESPONSE_CONTENT);
     }
 
+    @Override
+    public void writeResponseContentPreview(HttpData data) {
```

Comment:
`writeRequestContentPreview` and `writeResponseContentPreview` could be de-duplicated.